src/extraction/FStar.Extraction.ML.Util.fs

```diff
@@ -70,6 +70,7 @@ let mlconst_of_const' (sctt : sconst) =
   | Const_set_range_of ->
     failwith "Unhandled constant: range_of/set_range_of"
 
+  | Const_real _
   | Const_reify
   | Const_reflect _ ->
     failwith "Unhandled constant: reify/reflect"
```

Comment:
Change the error message to be inclusive of `Const_real` too?

Reply:
done